templates/repo/home.tmpl

```diff
@@ -28,34 +28,33 @@
 				</div>
 			{{end}}
 		</div>
-		<div class="gt-mt-3" id="repo-topics">
-		{{range .Topics}}<a class="ui repo-topic large label topic" href="{{AppSubUrl}}/explore/repos?q={{.Name}}&topic=1">{{.Name}}</a>{{end}}
-		{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}<a id="manage_topic" class="muted">{{.locale.Tr "repo.topic.manage_topics"}}</a>{{end}}
+		<div class="gt-df gt-ac gt-fw gt-mt-3" id="repo-topics">
+			{{range .Topics}}<a class="ui repo-topic large label topic" href="{{AppSubUrl}}/explore/repos?q={{.Name}}&topic=1">{{.Name}}</a>{{end}}
+			{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}<button id="manage_topic" class="ui button tiny tertiary gt-ml-2">{{.locale.Tr "repo.topic.manage_topics"}}</button>{{end}}
```

Comment:
Can we avoid using `.ui .tertiary.button`? There is only one single case of it in the whole repo, and I prefer we lessen our Fomantic dependency, not extend it.

Reply:
Of course, if you can provide a correct button-like element. 

It needs:

* button-like (at least, role=button)
* handles Space & Enter

Reply:
Please show a screenshot how it currently looks.

Reply:
A button that renders as a link? I'd argue that's not something we should do as it's confusing UX. Does a `basic` button not work there? That's why I asked for a screenshot.

Reply:
As before without underline. IIRC basic button has border.

Reply:
![image](https://user-images.githubusercontent.com/2114189/226107696-ba793f0a-2acb-4d29-befd-494cec501708.png)

![image](https://user-images.githubusercontent.com/2114189/226107706-7d50239b-7c58-4b58-beed-88108b7eacf1.png)

Reply:
I meant a screenshot of how it looks with this PR, as I'm too lazy currently to try this PR 😉

Reply:
I guess there are only slight color&font-size difference due to `a` and `.ui.button.tiny..... `

Before:

![image](https://user-images.githubusercontent.com/2114189/226150200-5a5deb29-02c6-4476-8cf3-5c3f36ce91ec.png)

After:

![image](https://user-images.githubusercontent.com/2114189/226150275-ad6051d6-2846-4d4f-baa3-0bb8ead76082.png)

Reply:
Ah, I understand now why a button does not fit there as it looks too similar to the labels.